### PR TITLE
New package: lldpd-1.0.5

### PR DESCRIPTION
**Pull request by zdykstra**

### CODE REVIEW

**Comment by abenson** [comment]
Does it need to be told somewhere to use the created accounts?

**Comment by zdykstra** [comment]
> Does it need to be told somewhere to use the created accounts?

It automagically privseps to that account by default. 

```
ps auwx | grep lldpd
root      2282  0.0  0.0   2616   920 ?        Ss   May25   0:03 runsvdir -P /run/runit/runsvdir/current log: ] protocol CDPv2 enabled 2020-05-30T00:15:01 [INFO/main] protocol SONMP disabled 2020-05-30T00:15:01 [INFO/main] protocol EDP disabled 2020-05-30T00:15:01 [INFO/main] protocol FDP disabled 2020-05-30T00:15:01 [INFO/event] libevent 2.1.11-stable initialized with epoll method 2020-05-30T00:15:01 [INFO/lldpctl] lldpd should resume operations ......................................................
root     27464  0.0  0.0   2464   648 ?        Ss   00:14   0:00 runsv lldpd
_lldpd   28630  0.0  0.0   4344  2296 ?        S    00:15   0:00 lldpd -d -c
_lldpd   28632  0.0  0.0   3928  1408 ?        S    00:15   0:00 lldpd -d -c
```

File: srcpkgs/lldpd/files/lldpd/run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec lldpd ${OPTS:- -d}
```
Comment (CameronNemo) [inline srcpkgs/lldpd/files/lldpd/run:3]: this logs to stderr, no? which would leak up to runsvdir? Probably want to redirect to stdout and set up a logger.

Comment (zdykstra) [reply srcpkgs/lldpd/files/lldpd/run]: Good catch. I've redirected stderr to stdout and linked in vlogger. There seemed to be three distinct ways to do logging, so I'm not sure if this is the preferred method or not.

**Comment by zdykstra** [comment]
Re-based onto master!

**Pull request merged**